### ISSUE TITLE
When citation is on multiple lines, only the span of the citation on the last line is highlighted

### ISSUE THREAD

**Issue by andrewhead** [allenai/scholarphi#25]
The ideal behavior is that the spans for a citation would be highlighted on all lines. This is likely due to limitations in the current code that only adds an annotation for the first bounding box for each entity. This might be one that's quick to fix.

**Comment by andrewhead** [comment]
Addressed in https://github.com/allenai/scholar-reader/commit/c6328ec313cdbf8c50bc923bab6bf0d6b0f45068, https://github.com/allenai/scholar-reader/commit/0e558b42cb2d3cffea8752776d8b2d767bc5ce89, and https://github.com/allenai/scholar-reader/commit/70ec189dc97239b6d7f4b67705c8d9992b840b3b.

There remains two tasks to satisfy this issue:
- [ ] I think I introduced latency issues. Test the reader to check if I indeed slowed down the performance and, if so, fix it up to have similar performance to before.
- [ ] When there's a multi-line citation and someone hovers over the text, it should highlight all parts of the multi-line citation
